tests/main/views/test_buyers.py

```diff
@@ -1692,189 +1692,241 @@ def test_show_draft_brief_summary_page(self, data_api_client):
                 'How to shortlist suppliers',
                 'How to evaluate suppliers',
                 'How to award a contract',
-                'View the Digital Outcomes and Specialists contract',
+                'Download the Digital Outcomes and Specialists contract',
             ]
 
+            assert "Awarded to " not in page_html
             assert document.xpath('//a[contains(text(), "Delete")]')
 
-    def test_show_live_brief_summary_page_for_live_and_expired_framework(self, data_api_client):
-        framework_statuses = ['live', 'expired']
+    @pytest.mark.parametrize('framework_status', ['live', 'expired'])
+    def test_show_live_brief_summary_page_for_live_and_expired_framework(self, data_api_client, framework_status):
         with self.app.app_context():
             self.login_as_buyer()
-            for framework_status in framework_statuses:
-                data_api_client.get_framework.return_value = api_stubs.framework(
-                    slug='digital-outcomes-and-specialists',
-                    status=framework_status,
-                    lots=[
-                        api_stubs.lot(slug='digital-specialists', allows_brief=True),
-                    ]
-                )
-                brief_json = api_stubs.brief(status="live")
-                brief_json['briefs']['publishedAt'] = "2016-04-02T20:10:00.00000Z"
-                brief_json['briefs']['specialistRole'] = 'communicationsManager'
-                brief_json['briefs']["clarificationQuestionsAreClosed"] = True
-                data_api_client.get_brief.return_value = brief_json
+            data_api_client.get_framework.return_value = api_stubs.framework(
+                slug='digital-outcomes-and-specialists',
+                status=framework_status,
+                lots=[
+                    api_stubs.lot(slug='digital-specialists', allows_brief=True),
+                ]
+            )
```

Comment:
We should make this `api_stubs.brief` take `**kwargs` at some point and update the brief dict off that. Would be super helpful. Then this would just be:
```
api_stubs.brief(
    status="live",
    publishedAt="2016-04-02T20:10:00.00000Z",
    specialistRole='communicationsManager',
    clarificationQuestionsAreClosed=True
)
```
Out of scope of this pr though

Reply:
I can add a tech debt ticket for this?

Reply:
Which is now here: https://trello.com/c/TzpZsdVg/176-api-client-apistubs-should-accept-kwargs-to-simplify-test-setups

Reply:
Ah nice one